sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_v1/query.sql

```diff
@@ -0,0 +1,108 @@
+WITH distinct_countries AS (
+  -- Some country codes appear multiple times as some countries have multiple names.
+  -- Ensure that each code appears only once and go with name that appears first.
+  SELECT
+    code,
+    name
+  FROM
+    (
+      SELECT
+        row_number() OVER (PARTITION BY code ORDER BY name) AS rn,
+        code,
+        name
+      FROM
+        `moz-fx-data-derived-datasets`.static.country_names_v1 country_names
+    )
+  WHERE
+    rn = 1
+),
+pop AS (
+  SELECT DISTINCT
```

Comment:
Why are we using SELECT DISTINCT here? If the intention is to make sure that we only get one entry per client, this doesn't accomplish that. If a client were to send two new_profile pings with any difference in any of the fields listed here, they would both be in the result set.

I'd prefer to see this expressed with ROW_NUMBER() like in the distinct_countries CTE above.